lms/resources/_js_config/__init__.py

```diff
@@ -68,9 +68,20 @@ def add_document_url(self, document_url):
                 ),
             }
         elif document_url.startswith("vitalsource://"):
+            lti_params = self._context.lti_params
             vitalsource_svc = self._request.find_service(name="vitalsource")
-            self._config["vitalSource"] = {
-                "launchUrl": vitalsource_svc.get_launch_url(document_url)
+            book_location = vitalsource_svc.parse_document_url(document_url)
+            self._config["api"]["viaUrl"] = {
```

Comment:
We should changing the `viaUrl` naming here so we can reuse it for assignments where the launch steps are the same, but Via is not involved.

Reply:
This is handled in https://github.com/hypothesis/lms/pull/4160.